### PR TITLE
NEW: DFBR Registration

### PR DESCRIPTION
**Pull request by ruqayya**
DFBR registration

### CODE REVIEW

File: tiatoolbox/tools/registration/wsi_registration.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import torchvision
+from torchvision.models._utils import IntermediateLayerGetter
+
+
+class rigidRegistration:
```
Comment (shaneahmed) [inline tiatoolbox/tools/registration/wsi_registration.py:5]:
```suggestion
class RigidRegistration:
```

tiatoolbox/tools/registration/wsi_registration.py:5:8: N801 class name 'rigidRegistration' should use CapWords convention

File: tiatoolbox/tools/registration/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+"""Models package for the models implemented in tiatoolbox."""
```
Comment (shaneahmed) [inline tiatoolbox/tools/registration/__init__.py:1]:
```suggestion
"""Models package for the models implemented in tiatoolbox."""
from tiatoolbox.tools.registration import wsi_registration
```

File: tiatoolbox/tools/registration/wsi_registration.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    def __init__(self):
+        self.patch_size = (224, 224)
+        self.Xscale, self.Yscale = [], []
+        model = torchvision.models.vgg16(True)
+        return_layers = {"16": "block3_pool", "23": "block4_pool", "30": "block5_pool"}
+        self.FeatureExtractor = IntermediateLayerGetter(
```
Comment (shaneahmed) [inline tiatoolbox/tools/registration/wsi_registration.py:11]: You need to write corresponding test files to add coverage.

Comment (John-P) [inline tiatoolbox/tools/registration/wsi_registration.py:11]:
```suggestion
        self.feature_extractor = IntermediateLayerGetter(
```

Comment (John-P) [reply tiatoolbox/tools/registration/wsi_registration.py:11]: Following PEP8 naming rules.

File: tiatoolbox/tools/registration/wsi_registration.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import torchvision
+from torchvision.models._utils import IntermediateLayerGetter
+
+
+class rigidRegistration:
+    def __init__(self):
+        self.patch_size = (224, 224)
+        self.Xscale, self.Yscale = [], []
```
Comment (John-P) [inline tiatoolbox/tools/registration/wsi_registration.py:8]:
```suggestion
        self.x_scale, self.y_scale = [], []
```

Comment (John-P) [reply tiatoolbox/tools/registration/wsi_registration.py:8]: PEP8 naming rules.

File: tiatoolbox/tools/registration/wsi_registration.py (hunk 1 above)
Comment (shaneahmed) [reply tiatoolbox/tools/registration/wsi_registration.py]: Resolve in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/372/commits/0a9fb74a704756ba8c2c5512de29ccd5b864674a

File: tiatoolbox/tools/registration/__init__.py (hunk 2 above)
Comment (shaneahmed) [reply tiatoolbox/tools/registration/__init__.py:1]: Resolved in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/372/commits/b11fde44306914341e2853647df36cb8dadbd0fc

File: .idea/other.xml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (shaneahmed) [inline .idea/other.xml:1]: Need to remove this file from git. Add to gitignore.

File: .idea/vcs.xml [hunk 6]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (shaneahmed) [inline .idea/vcs.xml:1]: Need to remove this file from git. Add to gitignore.

**Pull request closed without merging**